nixos/modules/services/hardware/throttled.nix

```diff
@@ -8,6 +8,12 @@ in {
   options = {
     services.throttled = {
       enable = mkEnableOption "fix for Intel CPU throttling";
+
+      extraConfig = mkOption {
+        type = types.str;
+        default = "";
+        description = "Alternative configuration";
```

Comment:
Are there any upstream docs you can link to?

Reply:
Nope only the repository README